### PR TITLE
DOCS: describe anaconda release process

### PR DESCRIPTION
**Pull request by jkowalleck**
after the anaconda release was announced (see https://github.com/CycloneDX/cyclonedx-python-lib/issues/44#issuecomment-989912153)
we might need a shield for this.

here is the shield & some docs

### CODE REVIEW

**Comment by jkowalleck** [comment]
@madpah I am uncertain that the path i put the docs in are right.
could you check and advice where raw docs should be put in?

**Comment by jkowalleck** [comment]
docs target need to be adjusted/incorporated into structure after #120 was merged to master.

So i will pot this one as a "draft" until the 1.4 release is prepared.

**Review by madpah — CHANGES_REQUESTED** [review]
`doc/processes/release/conda-forge.md` will need to be updated to:
1. Be written in RST (ideally)
2. Relocated to `docs/...` and added to the `index.rst` or other files `toctree`

**Comment by jkowalleck** [comment]
relocated to `docs/` dir. 
did not link anywhere, as it is internal documentation.

File: docs/install.rst [hunk 1]
```diff
@@ -1,8 +1,12 @@
 Installation
 ============
 
-Install from pypi.org as you would any other Python module using your preferred package manager:
+Install from `pypi.org <https://pypi.org/project/cyclonedx-python-lib/>`
```
Comment (jkowalleck) [inline docs/install.rst:4]: tried to do as described here: https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#external-links

Comment (madpah) [reply docs/install.rst:4]: Should be like this @jkowalleck:

```
Install from `pypi.org`_.

.. _pypi.org: https://pypi.org/project/cyclonedx-python-lib/
```

**Pull request merged**